index.js

```diff
@@ -297,6 +298,48 @@ Blockchain.prototype.getBlock = function (hash, cb) {
 }
 
 /**
+ *Gets a block by its hash
+ * @method getBlocks
+ * @param blockId - the block's hash or number
+ * @param {Function} cb - the callback function
+ */
+Blockchain.prototype.getBlocks = function (blockId, maxBlocks, skip, reverse, cb) {
```

Comment:
Not sure why but eth62's GetBlockHeaders has a skip option https://github.com/ethereum/wiki/wiki/Ethereum-Wire-Protocol#new-model-syncing-pv62